src/hooks/useBreakpoints.ts

```diff
@@ -20,9 +21,8 @@ export default function () {
   }, [])
 
   return {
-    iPhoneSizes: width > smSize && width < 400,
-    xxs: width > xsSize,
-    xs: width > xsSize && width <= smSize,
+    xxs: width > xxsSize,
+    xs: width > xxsSize && width <= xsSize,
```

Comment:
The `xs:` usually means higher than, maybe we should rename it to `fromXxsToXs`?

Reply:
I don't think it's a good idea. Just changed it to the proper value